lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShapeUI.cpp

```diff
@@ -321,13 +316,12 @@ bool ProxyShapeUI::select(MSelectInfo& selectInfo, MSelectionList& selectionList
   {
     if(hitSelected)
     {
-#if MAYA_API_VERSION < 201700
-      Qt::KeyboardModifiers modifiers = QApplication::keyboardModifiers();
-#else
-      Qt::KeyboardModifiers modifiers = QGuiApplication::keyboardModifiers();
-#endif
-      bool shiftHeld = modifiers.testFlag(Qt::ShiftModifier);
-      bool ctrlHeld = modifiers.testFlag(Qt::ControlModifier);
+      int mods;
+      MString cmd = "getModifiers";
+      MGlobal::executeCommand(cmd, mods);
+
+      bool shiftHeld = (mods % 2);
+      bool ctrlHeld = (mods / 4 % 2);
```

Comment:
Super pedantic point... I would prefer something simpler along the lines of:
```c++
bool shiftHeld = (mods & 1);
bool ctrlHeld = (mods & 4);
```
See the following compiler output: https://godbolt.org/g/VDJrtq

All compilers will replace the %2 with an &1, however the division by 4 introduces a shift&rotate instruction (sar). By simply testing the bit flag directly (in this case 1 << 2, or 4), it slightly trims down the number of instructions.